### PR TITLE
ref(suspect commits): Handle None in event frames

### PR DESCRIPTION
**Pull request by ceorourke**
I'm not sure how this happens, but sometimes `None` was in the `event_frames` list - this checks if the frame exists before doing `.get()` on it. 

Fixes https://sentry.sentry.io/issues/5088663511/

### CODE REVIEW

File: src/sentry/utils/committers.py [hunk 1]
```diff
@@ -255,7 +255,7 @@ def get_event_file_committers(
     munged = munged_filename_and_frames(event_platform, frames, "munged_filename", sdk_name)
     if munged:
         frames = munged[1]
-    app_frames = [frame for frame in frames if frame.get("in_app")][-frame_limit:]
+    app_frames = [frame for frame in frames if frame and frame.get("in_app")][-frame_limit:]
```
Comment (ceorourke) [inline src/sentry/utils/committers.py:258]: Another option would be to remove None from the result of https://github.com/getsentry/sentry/blob/master/src/sentry/utils/event_frames.py#L209 but I was having trouble setting up a test for this, and ultimately they do the same thing.

**Review by schew2381 — COMMENTED** [review]
I think this is ok, but should get another pair of eyes on this. Don't know much about frames

**Comment by ceorourke** [comment]
> I think this is ok, but should get another pair of eyes on this. Don't know much about frames

lol I dont either, was just trying to knock out the stuff that's assigned to us

**Review by schew2381 — APPROVED** [review]
lgtm for now to prevent the issue

**Pull request merged**